study.md

```diff
@@ -22,7 +22,7 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, define ORM and explain why using an ORM is valuable.
 
 ```md
-<!-- your response here -->
+If I understand orm correctly it more or less represents what the data is in transit between objects to tables in a database. It is the variable that stores the data between useages.
```

Comment:
Object relational-mapping (ORM) is a method of representing arbitrary data as objects in a programming language.  They are valuable because they provide an easy-to-use API for database-related tasks.